index.html

```diff
@@ -0,0 +1,348 @@
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="card456">
+      <div class="desk-card-d1">
+        <h2 class="desk-prof-art-dash">Profesional Art Printing Data More</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+
+      <div class="desk-card-d2">
+        <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+
+      </div>
+
+      <div class="desk-card-w1">
+        <h2 class="desk-prof-art-w">Website Protfolio </h2>
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="section-3">
+    <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+        website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+    </div>
+    <hr class="line-3">
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
+      <img class="abt-img-desk" src="./assets/deskimage/Groupme.png" alt="">
```

Comment:
- [ ] As requested by previous reviewer you should implement and use on background image as decorations are always backgrounded images, not img tags).

Reply:
Dear @roshan-bajgain I am working mainly on mobile version of my portfolio but your giving me feedback on desktop version of my portfolio which is next week's project

Reply:
@solog0039, Below 768px we consider the design to be responsiveness, Please try to implement that